### PR TITLE
Change Bulk NFT Mint Pricing

### LINKED ISSUE #7769: Update mint pricing
Update prices of minting NFTs as described in this table:

Hedera Token Service Transaction | Previous Fee Schedule | New Fee Schedule
-- | -- | --
Mint a single NFT in a collection | $0.05 | $0.02
Bulk mint 10k NFTs in a collection | $76.80 | $200

Public announcement: https://hedera.com/blog/hederas-expanding-nft-ecosystem-new-pricing-to-meet-growing-demand

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Fixes #7769 

- Changes nft mint base price to $0.02 from $0.05
- Changed the calculation of bulk minting nfts based on number of serial numbers (numOfSerials * 0.02)
- Added EET validating price

Updated fee calculator : https://github.com/hashgraph/hedera-fee-tool-js/pull/132

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
LGTM! 🥂

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/AccessorBasedUsagesTest.java [hunk 1]
```diff
@@ -283,7 +284,8 @@ void worksAsExpectedForTokenMint() {
 
         subject.assess(sigUsage, txnAccessor, accumulator);
 
-        verify(tokenOpsUsage).tokenMintUsage(sigUsage, baseMeta, tokenMintMeta, accumulator);
+        verify(tokenOpsUsage)
+                .tokenMintUsage(eq(sigUsage), eq(baseMeta), eq(tokenMintMeta), eq(accumulator), any(SubType.class));
```
Comment (mhess-swl) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/AccessorBasedUsagesTest.java:288]: Nit:
```suggestion
                .tokenMintUsage(eq(sigUsage), eq(baseMeta), eq(tokenMintMeta), eq(accumulator), notNull(SubType.class));
```

Looking at how `SubType` is used, seems like `notNull(...)` makes more sense

**Review by mhess-swl — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/AccessorBasedUsagesTest.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/AccessorBasedUsagesTest.java:288]: yes, will do in next PR . Thanks @mhess-swl

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**